libdnf.spec

```diff
@@ -13,7 +13,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.0
+Version:        0.7.1
```

Comment:
@Conan-Kudo Please I try to solve a problem - mark required minimal version of libdnf that will provide the functionality. I cannot change in libdnf release from 0. because it is not released and it is not according to guidelines. But according to you I also cannot change version. Well we are in dead-end. Or do you have any other solution? I am really open to hear it, but ignoring potential problem is not a solution for me.